app/js/templates/app_view.ejs

```diff
@@ -1,4 +1,4 @@
-<p>
-  <span>Welcome</span>
-  <span>The current date is: <%= view.currentDate() %></span>
-</p>
\ No newline at end of file
+<div>
```

Comment:
Using a form with a submit button will eliminate the need to handle keyboard events on the textbox as presssing enter will automaticly submit the form.